src/bunit.web/TestDoubles/WebAssemblyHostEnvironment/FakeWebAssemblyHostEnvironment.cs

```diff
@@ -0,0 +1,49 @@
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+
+namespace Bunit.TestDoubles;
+
+/// <summary>
+/// Represents a fake <see cref="IWebAssemblyHostEnvironment"/> that makes the <see cref="Environment"/> and <see cref="BaseAddress"/> settable.
+/// </summary>
+public class FakeWebAssemblyHostEnvironment : IWebAssemblyHostEnvironment
+{
+	/// <summary>
+	/// Gets the name of the environment. This is configured to use the environment of
+	/// the application hosting the Blazor WebAssembly application. Configured to "Production"
+	/// when not specified by the host.
+	/// In the <see cref="FakeWebAssemblyHostEnvironment"/> it can also be set.
+	/// </summary>
```

Comment:
```suggestion
	/// <summary>
	/// Gets or sets the name of the environment. Default is <c>Production</c>.
	/// </summary>
```

I think this is sufficient.